README.md

```diff
@@ -17,6 +17,20 @@ A highly opinionated WordPress Pro scaffold, built with ❤️ by [GRRR](https:/
 
 View the [prerequisites](https://github.com/grrr-amsterdam/wordpress-scaffold/wiki/Prerequisites) in the wiki for more details.
 
+## Run with Docker
+
+Beside using a local webserver you can use Docker Compose to start a webserver with a database to run your WordPress Scaffold project.
+
+Build, create and start the containers:
+
+```shell script
+docker-compose up
+```
+
+Now visit [http://wordpress-scaffold.localhost](http://wordpress-scaffold.localhost) to see the result.
```

Comment:
Since this is currently before the quick start I'm not sure if there will be a real result there? 😄  (or would it show the default Apache webpage?)

Reply:
You're right. Docker must be started between create-project and setup. So it's necessary to bother the quick start with Docker information. I was hoping to prevent that.